src/components/logs/forms/medication-setup.tsx

```
@@ -0,0 +1,217 @@
+                                        {error?.message}
+                                    </div>
+                                ))}
+                            </>
+                        )}
+                    />
+                </div>
+            </div>
+            <div>
+                <form.Subscribe
+                    selector={(state) => [state.canSubmit, state.isSubmitting]}
+                    children={([canSubmit, isSubmitting]) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)